internal/configuration/config.go

```diff
@@ -35,6 +35,9 @@ type Config struct {
 	// list of Wavefront sinks. At least 1 is required.
 	Sinks []*WavefrontSinkConfig `yaml:"sinks"`
 
+	// list of Wavefront sinks used by envent listener.
+	EventSinks []*WavefrontSinkConfig `yaml:"events_sinks"`
```

Comment:
Let's just fold this into the existing Sinks.

Reply:
We should have a top level boolean config `collectEvents` that enables event collection and defaults to false.